### PR TITLE
Engagement/jgilmore/ak additional transform changes

### PR DESCRIPTION
**Pull request by GilmoreA6**
This PR updates the AK receiver transform to use the getCodingSystemMapping() function for the PID-10-3 field.

Test Steps:
1. run ./gradlew test
2. run ./gradlew testIntegration

## Changes
- adds element  ak-patient-race-coding-system to AK receiver transform that uses getCodingSystemMapping() custom FHIR function .

### Testing
- [x ] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?

### CODE REVIEW

File: prime-router/metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml
Comment (adegolier) [inline prime-router/metadata/hl7_mapping/receivers/STLTs/AK/AK-receiver-transform.yml:37]: Is there supposed to be quotes around this? The other ones don't seem to have it.

**Review by victor-chaparro — APPROVED** [review]
Approved unsure if Angela's comment needs to be resolved.

**Review by oslynn — APPROVED** [review]

**Review by adegolier — APPROVED** [review]
Tests work. Still crying about the inconsistency but that's already all over so I don't think it should hold this up.

**Pull request merged**
